src/freenas/usr/local/lib/middlewared_truenas/plugins/truenas.py

```diff
@@ -119,6 +119,15 @@ async def get_chassis_hardware(self):
         # Give up
         return 'TRUENAS-UNKNOWN'
 
+    @accepts()
+    async def get_chassis_series(self):
```

Comment:
I agree with Caleb. I dont see the need for this.

We can just do truenas.get_chassis_hardware == TRUENAS-M

Fix get_chassis_hardware to not return MINI